### ISSUE TITLE
Track and display broken achievements

### ISSUE THREAD

**Issue by bartervg** [bartervg/barter.vg#222]
# What problem does this feature address?
The site tracks number of achievements for Steam games. However, an estimated 6% of games have broken achievements [1] and there are over 30 thousand games with achievements [2]. Which games have broken achievements? For those that acquire games with the intent to receive 100% of achievements, this is useful information. 

## Describe a solution
Next to the achievement column, create a broken achievement column. Display this wherever achievement information is shown. Unfortunately, many of the broken achievement lists are no longer maintained. Astats does have a recently broken and fixed list though.

## Examples of similar features
Non-marketable card sets had to be tracked prior to Steam-tracker `banned` status.

**Comment by arseny92** [comment]
> Unfortunately, many of the broken achievement lists are no longer maintained. 

Doesn't [TrueSteamAchievements](https://truesteamachievements.com/) do it?

**Comment by bartervg** [comment]
Yes, it does look like they track broken and unobtainable achievements https://truesteamachievements.com/game/Mad-Max/achievements?unobsdiscos=1 but I don't see a way to easily access this information. In addition, the URLs use titles rather than appIDs. 

The initial challenge will be getting the estimated 1800 broken achievement games into the database. An API would help, but more laborious means are possible.

**Comment by antigravities** [comment]
Tracking broken achievements feels like feature creep for a Web site about *trading* video games. Sure, whether a game has achievements, and (as a stretch) how many achievements there are could be helpful to the average member, but I don't think I can say that I've met more than one person who cares much about broken achievements, and there are many resources for discovering this information for the one or two people that need it.

I'm not saying that this feature wouldn't have a place on Barter, but is developing and maintaining it really worth it in the long run?

**Comment by bartervg** [comment]
It is scope creep, but there is the possibility that it could be maintained with minimum effort. Although I spent a lot of time fixing cards and achievements, so `could be minimum effort` does seem like scope creep enabling.

The extent of the reporting would be similar to astats achievement count 
![image](https://user-images.githubusercontent.com/28576132/106171468-8d93e780-615f-11eb-8c48-29ac6e77cdb2.png). Achievements link to astats already, so the extra information would be the negative number. This would appear in offers, bundles and game profiles.

**Comment by bartervg** [comment]
I thought of a better justification / description of this feature.

This won't be an attempt to reproduce the work of Astat. No original research needed. I'm not going to download 1800 games and start testing each achievement. In the same way that Barter.vg isn't an achievement, delisted, review, or trading card site, but still provides the basic information and links to the definitive sources for more information. Having the overview information in the database allows for more informative links. Rather than `click here to check if this game has trading cards` `click here to check if it has achievements`. 

Yet, the question remains if enough people care. I think there is more than one person that cares about broken achievements. I'll document effort spent, and worse case scenario, this will be a lesson learned in scope management.
